### PR TITLE
[SYSTEMML-540] Support loading of batch normalization weights

### PR DESCRIPTION
**Pull request by niketanpansare**
To test this PR, please use resnet's deploy proto file and corresponding caffemodel:
```python
import systemml as sml
sml.convert_caffemodel(sc, 'resnet.proto', 'resnet.caffemodel', 'trained_resnet_weights1')
```

- This PR ignores reading of scalar moving average fraction in batchnorm layer.

@dusenberrymw

### CODE REVIEW

**Comment by akchinSTC** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/2083/

**Comment by akchinSTC** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/2084/

**Comment by akchinSTC** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/2092/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='SystemML-PullRequestBuilder/org.apache.systemml:systemml' /><a href='https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/2092/org.apache.systemml$systemml/testReport'>SystemML-PullRequestBuilder/org.apache.systemml:systemml</a>: <span class='status-failure'>1</span></h3><ul><li><a href='https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/2092/org.apache.systemml$systemml/testReport/org.apache.sysml.test.integration.functions.indexing/RightIndexingMatrixTest/testRightIndexingDenseCP/'><strong>org.apache.sysml.test.integration.functions.indexing.RightIndexingMatrixTest.testRightIndexingDenseCP</strong></a></li></ul>

**Comment by akchinSTC** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/2102/
